modules/command/dh/dh-replication-import-command.js

```diff
@@ -80,17 +80,13 @@ class DhReplicationImportCommand extends Command {
         // TODO: Verify distribution keys and hashes
         // TODO: Verify data creator id
 
-        if (privateData && Object.keys(privateData).length > 0) {
+        if (permissionedData && Object.keys(permissionedData).length > 0) {
```

Comment:
This is duplicated code from dh-data-read-request-free-command.js it's a good  example what can be placed in permissioned-data-service class